programs/stake/src/stake_instruction.rs

```diff
@@ -125,19 +121,6 @@ declare_process_instruction!(Entrypoint, DEFAULT_COMPUTE_UNITS, |invoke_context|
             )?;
             instruction_context.check_number_of_instruction_accounts(5)?;
             drop(me);
-            if !invoke_context
-                .feature_set
-                .is_active(&feature_set::reduce_stake_warmup_cooldown::id())
-            {
-                // Post feature activation, remove both the feature gate code and the config completely in the interface
```

Comment:
I can't remember why we thought this would be possible... the configuration is not at the end of the account list, so we need to have *some* account at index 4 when we create the `delegate_stake` instruction:

https://github.com/anza-xyz/agave/blob/e261e2704a1a874b4df5802cbeb6d119d3dc4240/sdk/program/src/stake/instruction.rs#L104-L117

At the very least, could you update the comment there to say something like "unused account, formerly the stake config"? And how about updating the instruction creator to use `system_program::id()` or something?